client/src/App.js

```diff
@@ -51,11 +52,12 @@ function App() {
             <Route path="/faq" component={FAQ} />
             <Route path="/blog" component={Blog} />
             <Route path="/profile" component={Profile} />
+			<Route path="/interview" component={Interview} />
```

Comment:
I think with the new linter changes, lint this page, and that'll fix the spacing issue here :)

Reply:
Whoops! My bad.